### PR TITLE
Upgrade to firebase crashlytics

### PR DESCRIPTION
**Pull request by lucasvanhalst**

### CODE REVIEW

File: rogerthat/build.gradle [hunk 1]
```diff
@@ -122,9 +119,13 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:17.0.0'
     implementation 'com.google.android.gms:play-services-maps:17.0.0'
     implementation 'com.google.maps.android:android-maps-utils:1.2.1'
-    implementation 'com.google.firebase:firebase-core:17.3.0'
+    implementation 'com.google.firebase:firebase-analytics:17.4.1'
```
Comment (RubenMattan) [inline rogerthat/build.gradle:122]: we don't need analytics

**Comment by RubenMattan** [comment]
@lucasvanhalst 

rogerthat/src/main/AndroidManifest.xml:75:13-60 Error:
	Attribute meta-data#firebase_crashlytics_collection_enabled@value at 
AndroidManifest.xml:75:13-60 requires a placeholder substitution but no value for <crashlyticsCollectionEnabled> is provided.
rogerthat/src/main/AndroidManifest.xml Error:
	Validation failed, exiting

File: rogerthat/build.gradle [hunk 2]
```diff
@@ -73,6 +70,10 @@ android {
                 checkReleaseBuilds false
                 abortOnError false
             }
+            firebaseCrashlytics {
```
Comment (lucasvanhalst) [inline rogerthat/build.gradle:73]: this isn't needed as it's the default

Comment (lucasvanhalst) [reply rogerthat/build.gradle:73]: actually nvm, I forgot to push a part of build.gradle

**Pull request merged**